packages/wallets/src/components/Base/ModalStepWrapper/ModalStepWrapper.tsx

```diff
@@ -27,17 +29,25 @@ const ModalStepWrapper: FC<PropsWithChildren<TModalStepWrapperProps>> = ({
                 'wallets-modal-step-wrapper--fixed': shouldFixedFooter,
             })}
         >
-            <div className='wallets-modal-step-wrapper__header'>
-                <WalletText size={isMobile ? 'sm' : 'md'} weight='bold'>
-                    {title}
-                </WalletText>
-                <CloseIcon className='wallets-modal-step-wrapper__header-close-icon' onClick={hide} />
-            </div>
+            {!shouldHideHeader && (
+                <div className='wallets-modal-step-wrapper__header'>
+                    {title && (
+                        <WalletText size={isMobile ? 'sm' : 'md'} weight='bold'>
+                            {title}
+                        </WalletText>
+                    )}
+                    <CloseIcon className='wallets-modal-step-wrapper__header-close-icon' onClick={hide} />
+                </div>
+            )}
             <div className='wallets-modal-step-wrapper__body'>
                 {children}
-                {!shouldFixedFooter && <div className='wallets-modal-step-wrapper__footer'>{renderFooter()}</div>}
+                {!shouldFixedFooter && renderFooter && (
```

Comment:
```suggestion
                {!shouldFixedFooter && renderFooter && (
```
Best to use `typeof renderFooter === 'function'` and save it in a variable